components/popup/readme.md

```diff
@@ -0,0 +1,25 @@
+# vwc-popup
+
+Popup are used to display a message or notification to the user and are displayed on top of all other web page elements.
+Popup's goal is to provide additional, helpful content.   
+To trigger the Popup, it should be paired with an anchor (e.g., a button).
+
+## Properties
+
+| Property      | Attribute     | Type                   | Default    |
+| ------------- | ------------- | ---------------------- | ---------- |
+| `anchor`      | `anchor`      | `Element \| undefined` |            |
+| `arrow`       | `arrow`       | `boolean \| undefined` | true       |
+| `corner`      | `corner`      | `Placement`            | "left"     |
+| `dismissible` | `dismissible` | `false \| undefined`   |            |
+| `distance`    | `distance`    | `number`               | 10         |
+| `open`        | `open`        | `boolean`              | false      |
+| `strategy`    | `strategy`    | `Strategy`             | "absolute" |
```

Comment:
is the strategy `absolute` by floating UI? would have figure it to be `fixed` 🤔

Reply:
Changed the default value to be fixed